io.github.mujx.Nheko.json

```diff
@@ -0,0 +1,210 @@
+          "path": "cmake/OlmConfig.cmake.in",
+          "type": "file"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "./configure --prefix=/usr",
+        "make && make check",
+        "make install"
+      ],
+      "buildsystem": "simple",
+      "name": "sodium",
+      "sources": [
+        {
+          "sha256": "eeadc7e1e1bcef09680fb4837d448fbdf57224978f865ac1c16745868fbd0533",
+          "type": "archive",
+          "url": "https://github.com/jedisct1/libsodium/releases/download/1.0.16/libsodium-1.0.16.tar.gz"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "./bootstrap.sh --with-libraries=random,thread,system,iostreams,atomic,chrono,date_time,regex --prefix=/usr",
+        "./b2 -d0 variant=release link=shared threading=multi --layout=system",
+        "./b2 -d0 install"
+      ],
+      "buildsystem": "simple",
+      "name": "boost",
+      "sources": [
+        {
+          "sha256": "5721818253e6a0989583192f96782c4a98eb6204965316df9f5ad75819225ca9",
+          "type": "archive",
+          "url": "https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.bz2"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "cmake -H. -Bbuild -DBUILD_LIB_TESTS=OFF -DBUILD_LIB_EXAMPLES=OFF -DCMAKE_BUILD_TYPE=Release",
+        "cmake --build build --target install --config Release"
+      ],
+      "buildsystem": "simple",
+      "name": "mtxclient",
+      "sources": [
+        {
+          "sha256": "184d155ec731a23750583c8a69a8cadda8af70994fea66c20f8231fb3072ddad",
+          "type": "archive",
+          "url": "https://github.com/mujx/mtxclient/archive/ca66424489b4d3e6ea447b853157fa88f0b849b3.tar.gz"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DTWEENY_BUILD_DOCUMENTATION=OFF -DTWEENY_BUILD_EXAMPLES=OFF",
+        "cmake --build build --config Release --target install"
+      ],
+      "buildsystem": "simple",
+      "name": "tweeny",
+      "sources": [
+        {
+          "sha256": "482857256a7235646004682912badb6521d361ed6987c8ebdae7986bf64ce694",
+          "type": "archive",
+          "url": "https://github.com/mobius3/tweeny/archive/43f4130f7e4a67c19d870b60864bc2862c19b81f.tar.gz"
+        }
+      ]
+    },
+    {
+      "build-commands": [
+        "cmake -H. -Bbuild -DCMAKE_BUILD_TYPE=Release -DLMDBXX_INCLUDE_DIR=.deps/lmdbxx",
+        "cmake --build build --config Release"
+      ],
+      "buildsystem": "simple",
+      "name": "nheko",
+      "post-install": [
+        "./install-resources.sh"
+      ],
+      "sources": [
+        {
+          "sha256": "bfc9d4e3715241c1878dbf16ff8cd5df18651e4dd32f780e0957d3724de57592",
+          "type": "archive",
+          "url": "https://github.com/mujx/nheko/archive/v0.5.2.tar.gz"
+        },
+        {
+          "path": "install-resources.sh",
+          "type": "file"
+        },
+        {
+          "path": "nheko.appdata.xml",
```

Comment:
As you are the upstream developer, why not keep it in your repo? This way other distributions can use it.

Reply:
It will be done in the next release, because the current `appdata.xml` files has a macos screenshot.